README.md

```diff
@@ -216,13 +217,13 @@ receiving server to persuarde the sending server to share the same resource with
 TODO: document how receiver.com can know if sender.com understood and processed the
 reshare request.
 
-### Multi Factor Authentication
+## Appendix A: Multi Factor Authentication
 If an OCM provider exposes the capability `/mfa-capable`, it indicates that it will try and comply with a MFA requirement set as a permission on a share. If the sharer OCM provider trusts the receiver OCM provider, the sharer MAY set the permission `mfa-enforced` on a share, which SHOULD be honored. A compliant OCM provider that signals that it is MFA-capable MUST not allow access to a resource protected with the `mfa-enforced` permission, if the consumer has not provided a second factor to establish their identity with greater confidence.
```

Comment:
Should we switch to `sending server` and `receiving server` in this section, to conform with new definitions?

Reply:
Yes! Done.